### PR TITLE
Prevent success message if no upgrade is needed

### PR DESCRIPTION
**Pull request by thardeck**

### CODE REVIEW

**Comment by suse-tests-pass** [comment]
Well Done! Your tests are still passing.
( Success: Merged 7963054c12b9e371083411eabc781369f521e840 into 005e5c6d8aeaa553ada500f06ee7d09a08f836e0 )

**Comment by suse-tests-pass** [comment]
Well Done! Your tests are still passing.
( Success: Merged d08f51d5a0e141b99ca647ea56d2cc0a7c101162 into 005e5c6d8aeaa553ada500f06ee7d09a08f836e0 )

File: lib/upgrade_format_task.rb [hunk 1]
```diff
@@ -54,7 +54,9 @@ def upgrade(store, name, options = {})
         Machinery::Ui.puts "No system descriptions were upgraded."
       end
     else
-      Machinery::Ui.puts "System description \"#{name}\" successfully upgraded."
+      if migrations_done > 0
```
Comment (aduffeck) [inline lib/upgrade_format_task.rb]: Do you mind making this an `elsif` instead of `else if`?

**Comment by aduffeck** [comment]
lgtm

**Pull request merged**